manifests/rhoai/shared/apps/pgvector/pgvector-app.yaml

```diff
@@ -0,0 +1,1188 @@
+        C30.406792,116.242676 24.888126,114.478790 19.167118,112.448792
+        C20.015089,109.332825 21.729599,108.588036 24.788712,109.534470
+        C29.211521,110.902817 33.812290,111.695969 38.692635,112.879280
+      z"/>
+      <path fill="#5E8C6C" opacity="1.000000" stroke="none"
+        d="
+      M35.606812,123.156982
+        C35.789543,124.792076 35.748646,126.549278 35.617722,128.740311
+        C35.175583,129.148468 34.823479,129.122818 34.208038,129.057007
+        C33.989399,127.562378 34.034088,126.107895 34.060936,124.335968
+        C34.489796,123.772049 34.936489,123.525574 35.606812,123.156982
+      z"/>
+      <path fill="#77ACB4" opacity="1.000000" stroke="none"
+        d="
+      M145.848999,70.807526
+        C145.640488,71.893135 145.330841,72.750092 144.826782,73.892654
+        C144.376465,72.624977 144.120544,71.071701 143.964890,69.178375
+        C144.626053,69.418503 145.186951,69.998680 145.848999,70.807526
+      z"/>
+      <path fill="#6D9EA9" opacity="1.000000" stroke="none"
+        d="
+      M162.809326,77.581665
+        C155.586060,83.251228 152.536377,92.720543 155.286331,102.256897
+        C155.286453,102.935188 155.064819,103.122307 154.597382,103.340233
+        C152.797211,104.178322 151.199066,104.933784 149.326782,105.883133
+        C149.792389,99.689644 149.381836,92.855095 151.635986,87.056404
+        C153.438217,82.420219 158.482010,79.044144 162.417145,75.031006
+        C162.773468,75.704140 162.799713,76.455063 162.809326,77.581665
+      z"/>
+      <path fill="#87C2C5" opacity="1.000000" stroke="none"
+        d="
+      M114.744354,115.271545
+        C115.926781,114.270096 116.862343,113.531731 117.936279,112.528961
+        C120.913597,109.396568 123.752525,106.528564 126.884804,103.524002
+        C127.262512,106.521317 128.432144,110.168182 127.236755,112.697166
+        C122.881470,121.911346 109.377533,130.211044 99.542740,118.024147
+        C95.714745,113.280632 92.004539,108.442047 88.124321,103.320633
+        C93.282715,106.951096 98.475616,111.023819 103.862679,114.821083
+        C107.136353,117.128662 110.768753,118.074013 114.744354,115.271545
+      z"/>
+      <path fill="#87C2C5" opacity="1.000000" stroke="none"
+        d="
+      M84.101219,97.322708
+        C85.296364,98.764511 86.587273,100.534584 87.942024,102.649307
+        C86.736237,101.212959 85.466614,99.431961 84.101219,97.322708
+      z"/>
+    </svg>
+  docsLink: https://www.enterprisedb.com/docs/edb-postgres-ai/ai-factory/vector-engine/
+  category: Self-managed
+  support: third party support
+  quickStart: ''
+  getStartedLink: https://validatedpatterns.io/patterns/rag-llm-gitops
+  getStartedMarkDown: >-
+    # Get started with EDB Postgres AI
+
+    [![Unleashing AI with Postgres, Episode 4](https://img.youtube.com/vi/nfsglUqDZnA/0.jpg)](https://www.youtube.com/watch?v=nfsglUqDZnA)
+
+    ## Quick links
+
+    - [Install](https://www.enterprisedb.com/contact): Install EDB AI Accelerator
```

Comment:
Not sure I understand this linking -- but I confirmed with Dana and he says this content is accurate.